src/tests/functional/plugin/conformance/test_runner/README.md

```diff
@@ -44,6 +44,7 @@ The target is able to take the following command-line arguments:
 * `--report_unique_name` allows to save report with unique name (report_pid_timestamp.xml). Mutually exclusive with --extend_report.
 * `--save_report_timeout` allows to try to save report in cycle using timeout (in seconds).
 * `--output_folder` Paths to the output folder to save report.
+* `--per_test_timeout` Setup timeout for each test in seconds, tests are not time-limited by default.
```

Comment:
`test_timeout`? It will be simplier. And Set any default value (for example, 15 min)